### DISCUSSION TITLE
docs/guide/webConsole/intro/

### DISCUSSION THREAD

**Discussion by giscus** [iPanelDev/docs#2] [Giscus]
# docs/guide/webConsole/intro/

Static Badge

https://ipaneldev.github.io/docs/guide/webConsole/intro/

**Comment by KelvinZhang512** [comment]
试问一下这个网页控制台的账号是什么？

**Reply by Zaitonn** [reply]
暂时没有用，可随便填（`v2.1`）

账号功能将在`v2.3`加入
